### ISSUE TITLE
Einddatum is een dag eerder (indien zaak wordt gestart en gesloten op dezelfde dag) dan gestuurd

### ISSUE THREAD

**Issue by joeribekker** [open-zaak/open-zaak#959]
*Gemeld door Martijn van Genetics*

**General information**
Open Zaak version: 1.3.5

**Describe the bug**
Wellicht gevolg van timezones. Opgestuurd wordr 26 april 00:00+02, en de response geeft 25 april 22:00+0. De einddatum wordt dan 25 april.

**To Reproduce**
1. ...
2. ...

**Expected behavior**

**Comment by sergei-maertens** [comment]
Yes, the reference implementation has this same constraint: https://github.com/VNG-Realisatie/zaken-api/blob/master/src/zrc/datamodel/models/core.py#L393

**Comment by joeribekker** [comment]
Refinement: To reproduce, I think you can just create a zaak with s`tartdatum=somedate+2:00` and see what the response returns.

**Comment by annashamray** [comment]
I've managed to reproduce it in one particular case:

If you create the closing status with `datumStatusGezet=2023-01-01 00:00:00+02:00`, then `zaak.enddatum` would be saved as `2022-12-31`

So the problem arises from the fact that we have `Zaak.startdatum` and `Zaakeinddatum` as dates and `Status.datumStatusGezet` as datetime. And when we close zaak we convert `Status.datumStatusGezet` to default timezone (UTC) and then take the date part from it. 

In my opinion the proper solution would be to use aware datetimes everywhere, but since it affects the ZGW standard it's not feasible. 
I'll use client timezone when saving `zaak.enddatum` in the bugfix
